specify7_ipup.py

```diff
@@ -23,10 +24,9 @@ def set_direct():
             none
             """
     # get current directory
-    #current_directory = os.getcwd()
+    # current_directory = os.getcwd()
     # this is absolute path for running chron from bin
-    current_directory = "/Users/mdelaroca/Documents/sandbox_db/" \
-                        "specify-sandbox/specify-asset-importer"
+    current_directory = os.path.dirname(os.path.abspath(__file__))
```

Comment:
changed to get filepath directory regardless of who's machine it is operating on